docs/doxygen/overviews/unicode.h

```diff
@@ -234,7 +234,7 @@ The problems can be separated into three broad classes:
 Because of the need to support implicit conversions to both @c char and
 @c wchar_t, wxString implementation is rather involved and many of its operators
 don't return the types which they could be naively expected to return.
-For example, the @c operator[] doesn't return neither a @c char nor a @c wchar_t
+For example, the @c operator[] returns neither a @c char nor a @c wchar_t
 but an object of a helper class wxUniChar or wxUniCharRef which is implicitly
 convertible to either. Usually you don't need to worry about this as the
```

Comment:
```suggestion
For example, the @c operator[] doesn't return either a @c char or a @c wchar_t
and instead returns an object of a helper class wxUniChar or wxUniCharRef that is implicitly
convertible to either. Usually you don't need to worry about this as the
```
I think this is also clearer in the other framing (either/or) with some modifications to the surrounding sentence.

Reply:
Thanks! I'll squash these changes into the first commit in a moment.